lesson17.md

```diff
@@ -0,0 +1,110 @@
+<!-- .slide: id="lesson17" -->
+
+# Basic Frontend - Spring 2020
+
+Lesson 17, Tuesday, 2020-05-05
+
+---
+
+### Iterating over object's keys
+
+Sometimes, we get a reply from a REST API like this:
+
+```js
+let translations = {
+    de: "Apfel",
+    fr: "Pomme",
+    tr: "Elma",
+    // ... and more translations
+}
+```
+
+How would we show the translations if we don't know the amount of translations in advance?
+
```

Comment:
can we also mention that "we might not know which languages (keys) exist in the REST API reply"?

Reply:
Good idea, I changed the phrasing